packages/preview/codly/0.1.0/typst.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "codly"
+version = "0.1.0"
+entrypoint = "codly.typ"
+authors = ["Dherse"]
+license = "MIT"
+description = "Codly is a beautiful code presentation template."
+repository = "https://github.com/Dherse/codly"
+exclude = ["LICENCE", "example/*"]
```

Comment:
demo.png should likely also be excluded and the LICENSE should definitely not be excluded. (not that it is right now because of typo :D)

Reply:
I might have missed it for some packages. The process should be more automated.